labs/practitioner-basics/4-monitoring/example-dashboards/lifecycle.json

```diff
@@ -8386,7 +8386,7 @@
             "type": "prometheus",
             "uid": "${DS_PROMETHEUS}"
           },
-          "expr": "sum by (partition) (irate(flink_taskmanager_job_task_operator_KafkaSourceReader_topic_partition_currentOffset{topic=\"cp4waiops_cartridge_lifecycle_input_alerts\"}[4h:30s]))",
+          "expr": "sum by (partition) (irate(flink_taskmanager_job_task_operator_KafkaSourceReader_topic_partition_currentOffset{topic=\"cp4aiops_cartridge_lifecycle_input_alerts\"}[4h:30s]))",
```

Comment:
You're probably right about this being namespace-driven, would be good to double-check these before committing.

Reply:
Yes, you are right. My latest commit leaves topic names as cp4waiops and all other references to namespace have been changed to cp4aiops.